package.json

```diff
@@ -36,6 +36,7 @@
     "express": "^4.17.1",
     "lorem-ipsum": "^2.0.3",
     "mysql2": "^2.2.5",
+    "nano": "^9.0.3",
```

Comment:
Just curious what this package is for?

Reply:
it is a package for interacting with CouchDB